README.md

```diff
@@ -94,7 +94,7 @@ We'd love for you to contribute to our open source projects. Before we can accep
 
 ## License
 ```
-Copyright (c) 2016 Uber Technologies, Inc.
+Copyright (c) 2017 Uber Technologies, Inc.
```

Comment:
We do not do this. This is usually set to the first/inception year that this project was opensourced